File: statistical_test/fishers_exact.R
#!/usr/bin/env Rscript
# 男性_生存、男性_死亡、女性_生存、女性_死亡
data = matrix(c(367, 1364, 344, 126), ncol=2)

print(fisher.test(data)$p.value) # 2.690694e-96

# 参考値：カイ二乗検定
# chisq.test(data)$p.value # => 7.565462e-101

File: statistical_test/fishers_exact.py
#!/usr/bin/env python
import scipy.stats as st # scipy version 1.3.1

data = [[367, 1364], [344, 126]]
print(st.fisher_exact(data)[1]) # => 2.6906937468576903e-96

# 参考値：カイ二乗検定
# st.chi2_contingency(data)[1] # => 7.565461964935766e-101
